dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx

```diff
@@ -157,8 +157,7 @@ export default function MaterialLabelSelector(props: LabelSelectorType) {
               renderTags={() => null}
               noOptionsText={
                 <div className={classes.noOption} onMouseDown={handleAddLabel}>
-                  Add new label
-                  <div className={classes.tag}> {truncate(labelValue, 10)}</div>
```

Comment:
@iamkryptonite Can you please explain why have you removed this?

Reply:
the tag class was for the labels that were selected and they had padding and margin assigned to them,which made the new label name shift downward. 

Before:
![1](https://user-images.githubusercontent.com/43510609/111901184-68836d00-8a5c-11eb-92e3-32fbb2a8c669.gif)
After:
![2](https://user-images.githubusercontent.com/43510609/111901221-9f598300-8a5c-11eb-9f84-912a10100820.gif)

Reply:
Okay